CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 Check out ways to contribute to Lion Web Components:
 
+## Feature requests
+
+When you have an idea on how we could improve, please check our [closed issues](https://github.com/ing-bank/lion/issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement) to see if there are similar ideas. If there are none, please [submit](https://github.com/ing-bank/lion/issues/new/choose) your feature request as an issue. Add a clear description of what you expect from the improvement and what the use case is. The issue will be closed and the community can add 👍 reactions to [vote and show interest](https://github.com/ing-bank/lion/issues?q=label%3A%22votes+needed%22+sort%3Areactions-%2B1-desc). When the feature gets picked up, the issue can be reopened.
```

Comment:
Change the link to: `https://github.com/ing-bank/lion/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Aenhancement+` ? So also currently worked on (open) features, can be found.

Reply:
I'll add `votes needed` labels to those as well. They might be closed again in the future. And your link would also show the `wont fix` or `fixed` issues.